decidim/decidim
# Redesign: datepicker

#### :tophat: What? Why?
Decidim version 28 got rid of the old Foundation datepicker, this pull request will add a new datepicker library to Decidim for "Date" -type input fields, with a custom timepicker for "Datetime-local" -type input fields.

This datepicker also supports US / North American style date/time formats which are at the moment used in a module we have developed: https://github.com/Pipeline-to-Power/decidim-module-ptp/tree/main/decidim-l10n

#### :pushpin: Related Issues
*Link your PR to an issue*
- Related to #10971 
- Fixes #10971 

#### Testing
1. Go to a meetings component (also available in other views)
2. Create a new meeting
3. See the start/end time fields displaying the date/time picker

### :camera: Screenshots
![image](https://github.com/decidim/decidim/assets/110532525/b169871d-d166-4d17-bed5-b78411d0d54c)

:hearts: Thank you!
